exercises/raindrops/src/example.c

```diff
@@ -1,25 +1,26 @@
 #include "raindrops.h"
 #include "stdio.h"
 #include "string.h"
+#include <stdlib.h>
 
-#define PLING_FACTOR 3
-#define PLANG_FACTOR 5
-#define PLONG_FACTOR 7
-
-char *convert(char *buffer, int buffer_length, int drops)
+char *convert(int drops)
 {
-   memset(buffer, '\0', sizeof(char) * buffer_length);
-   if ((drops % PLING_FACTOR != 0) &&
-       (drops % PLANG_FACTOR != 0) && (drops % PLONG_FACTOR != 0)) {
-      snprintf(buffer, buffer_length, "%d", drops);
-   } else {
-      snprintf
-          (buffer,
-           buffer_length,
-           "%s%s%s",
-           drops % PLING_FACTOR == 0 ? "Pling" : "",
-           drops % PLANG_FACTOR == 0 ? "Plang" : "",
-           drops % PLONG_FACTOR == 0 ? "Plong" : "");
+   // strlen("PlingPlangPlong") == 15, and max length of int is 12.
+   char *result = calloc(sizeof(char), 15);
```

Comment:
The tests fail in Travis because the maximum length of the string is _16_ not 15. there are 15 letters, plus the string terminator (the `'\0'` character).